src/internal/operators/publishReplay.ts

```diff
@@ -9,6 +9,85 @@ export function publishReplay<T>(bufferSize?: number, windowTime?: number, sched
 export function publishReplay<T, O extends ObservableInput<any>>(bufferSize?: number, windowTime?: number, selector?: (shared: Observable<T>) => O, scheduler?: SchedulerLike): OperatorFunction<T, ObservedValueOf<O>>;
 /* tslint:enable:max-line-length */
 
+/**
+ * Returns a ConnectableObservable, which is a variety of Observable that waits until its connect method is called
```

Comment:
This isn't _completely_ accurate. 

This "multicasts over a {@link ReplaySubject}", and that's the best high-level first sentence. 

Because it doesn't _always_ return a `ConnectableObservable`. If you pass a selector function to it (I talk about this below) it will return a normal `Observable`.